daemon/api.go

```diff
@@ -254,46 +254,67 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 	}
 	defer lock.Unlock()
 
-	sources := make([]string, 1, 2)
-	sources[0] = "local"
-	// we're not worried if the remote repos error out
-	found, _ := newRemoteRepo().All()
-	if len(found) > 0 {
-		sources = append(sources, "store")
+	results := make(map[string]map[string]interface{})
```

Comment:
This is a relatively complex thing (and/or I have a relatively simple mind). I wonder if we could have a small comment saying what is mapped to where? Maybe something like // map the source string to a map of package names to snap-data ?